chef/automate
# Fixing time zone being added to the selected calendar

### :nut_and_bolt: Description: What code changed, and why?
Fixing a bug in the calendar of when the first of the month is selected the calendar would switch to the previous month. 

The problem was that when the Date object is created it was adding the local time zone. 

### :+1: Definition of Done

When selecting the first day of the month on the calendar control on the compliance reporting search bar the month does not change. 

### :athletic_shoe: How to Build and Test the Change
1. `build components/automate-ui && start_all_services`
1. Open https://a2-dev.test/compliance/reports/overview
1. Open in the calendar control in the search bar and select the first day of the month. 
1. Ensure the month does not change on the calendar control. 

### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
